tests/PHPStan/Analyser/nsrt/bug-11129.php

```diff
@@ -68,7 +68,7 @@ public function foo(
 		assertType('non-falsy-string&numeric-string&uppercase-string', $float.$positiveInt);
 		assertType('non-falsy-string&uppercase-string', $float.$negativeInt);
 		assertType('non-falsy-string&uppercase-string', $float.$i);
-		assertType('non-falsy-string&uppercase-string', $i.$float); // could be 'non-falsy-string&numeric-string&uppercase-string'
+		assertType('non-falsy-string&uppercase-string', $i.$float);
```

Comment:
I don't think it `could be 'non-falsy-string&numeric-string&uppercase-string` since we could have a negative float.

So it will give something like `3-3.0` which is not a numeric string.